tests/test_fields.py

```diff
@@ -159,6 +169,20 @@ def test_deserialize_empty_relationship_node(self, post):
             field.deserialize({})
         assert excinfo.value.args[0] == 'Must include a `data` key'
 
+    def test_deserialize_many_non_list_relationship(self):
+        """Test deserializing many-to-one field with a non-list value."""
```

Comment:
I actually prefer not to have docstrings for tests; with pytest's output, it is easier to locate tests by method name.